Add datasets/ztf.yaml (IRSA-6717)

troyraen opened the pull request:
Add a yaml file for ZTF containing the metadata needed to generate an ODR webpage.

Note that there are 3 "Description" fields. The first describes ZTF in general and the length should remain <= 600 characters. The second describes the Objects Table and the third is Lightcurves. If/when we add Zubercal to the bucket we will add a separate description here for it.

Some information is still incomplete and has placeholder text in all caps. To do:

- [x] Given the name of this file, the url will be https://registry.opendata.aws/ztf/. Double check that that's what we want.
- [x] Find or create DOIs and add them here.
- [ ] Add the bucket name and prefixes once we know them.
- [ ] Double check 'RequesterPays' and 'AccountRequired' once the bucket has been set up.
- [x] Consider tutorials. I'm guessing that what we want to list here is the IRSA Tutorials page (as we have done for the other ODR datasets), but that does not currently contain any tutorials related to ZTF or HATS. I could add one that is based on LINCC's HATS tutorials fairly quickly. I'm guessing we do not want to list anything else here, but I'm not sure. Options would include Fornax notebooks and LINCC tutorials.

troyraen committed:
Add datasets/ztf.yaml

troyraen committed:
Add dataset format info

jkrick commented on datasets/ztf.yaml:
```diff
@@ -0,0 +1,32 @@
+License: https://irsa.ipac.caltech.edu/data_use_terms.html
+Citation: "If you use the Objects Table, please cite the Digital Object Identifier (DOI): [TODO](https://www.ipac.caltech.edu/doi/irsa/TODO). If you use the Lightcurves, please cite the Digital Object Identifier (DOI): [TODO](https://www.ipac.caltech.edu/doi/irsa/TODO). In addition, please follow the [ZTF acknowledgement guidelines](https://irsa.ipac.caltech.edu/data/ZTF/docs/releases/ztf_release ...
+Resources:
+  - Description: 'Objects Table is a catalog of PSF-fit photometry detections extracted from ZTF reference images. The reference images were generated by stacking single exposures acquired from all science programs in the survey, resulting in photometry up to 2.5 magnitudes deeper than single-exposure detections. Objects Table contains both point-like and extended objects. This version of the cat ...
```
```suggestion
  - Description: 'Objects Table is a catalog of PSF-fit photometry detections extracted from ZTF reference images. The reference images were generated by stacking single exposures acquired from all science programs in the survey, resulting in photometry up to 2.5 magnitudes deeper than single-exposure detections. Objects Table contains both point-like and extended objects.  The survey covers ~25,000 square degrees of the northern hemisphere.  This version of the catalog is in Apache Parquet format and partitioned following the Hierarchical Adaptive Tiling Scheme (HATS).'
```

jkrick replied:
sounds impressive.....

jkrick reviewed (approved):
1. It might be good to have a  reference to the HATS format.

2. For the tutorials comment above, I would suggest we do add a link to the Fornax light curve notebook which in relatively short order will have an example of accessing the ZTF HATS data from AWS.  I would not suggest working on a separate tutorial.

troyraen committed:
Add survey coverage

Co-authored-by: jkrick <jkrick@caltech.edu>

troyraen committed:
Patch formatting ztf.yaml

troyraen committed:
Add DOIs

troyraen committed:
Add HATS link

troyraen committed:
Add Fornax tutorial

troyraen commented:
I added the Fornax light curve notebook and a link to HATS docs. DOIs have been generated and I added them here. 

I think this is ready enough to open a PR upstream. There are a couple more to-dos and ultimately this will be blocked until the data is actually in the bucket, but an upstream PR will at least let the AWS folks (eg Chris) see that it's in progress. I will also email Chris again and let her know that the other steps are moving along. We do have an AWS account for this now, so hopefully I can log into that soon and get the bucket set up.

I will close this PR now in favor of a new one upstream, as that seems to work best with this forking workflow.

troyraen commented:
Superseded by https://github.com/awslabs/open-data-registry/pull/2639.
